### ISSUE TITLE
TODO list should be more redundant with issues

### ISSUE THREAD

**Issue by seirll** [calops/certifiedmakefile#9]
Currently the TODO list and the Issues are 2 separated things. We should handle that.

**Comment by maugier** [comment]
Good point. I just added a single line "Make TODO list redundant with the issues on GitHub" to the TODO list in my branch, so that at least this single point is itself redundant.
